fixes of i18n on readme.md (#184)
vtex-apps/search: docs/README.md

<!-- ALL-CONTRIBUTORS-BADGE:END -->

The VTEX Search app handles the new [**VTEX Intelligent Search**](https://help.vtex.com/tracks/vtex-intelligent-search) solution in IO stores by providing new UI components to enhance the search experience, such as the autocomplete feature.

![search-app-gif](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/vtex-search-0.gif)

## Configuration

> To function properly, Search app components require the `vtex.admin-search@1.x` and `vtex.search-resolver@1.x` apps to be installed in your store. For more information, read our [**VTEX Intelligent Search**](https://help.vtex.com/tracks/vtex-intelligent-search) documentation.

Reviewer: ```suggestion
The Search app requires the `vtex.admin-search@1.x` and `vtex.search-resolver@1.x` apps to be installed in your store. For more information, read the [Intelligent Search](https://help.vtex.com/tracks/vtex-intelligent-search) documentation.
```

| `search-banner`               | Renders a customized banner based on the search query.                                                                                                                                                                                                                                                                                                        |
| `did-you-mean`                | Helps users with possible misspelling corrections for the current search query.                                                                                                                                                                                                                                                                               |
| `search-suggestions`          | Renders a list of similar search terms for the current search query.                                                                                                                                                                                                                                                                                          |

> > ℹ️ The `search-banner`, `did-you-mean`, and `search-suggestions` blocks require [Search Result app](https://developers.vtex.com/docs/guides/vtex-search-result) version `3.x` or higher installed in your theme.

Reviewer: ```suggestion
> ℹ️ The `search-banner`, `did-you-mean`, and `search-suggestions` blocks require [Search Result app](https://developers.vtex.com/docs/apps/vtex.search-result) version `3.x` or higher installed in your theme.
```

### Adding Search blocks to the theme

> First, declare the `autocomplete-result-list.v2` block as a child block of the [`search-bar` block](https://developers.vtex.com/docs/guides/vtex-store-components-searchbar), exported by the Store Components app. Example:

Reviewer: ```suggestion
First, declare the `autocomplete-result-list.v2` block as a child block of the [`search-bar` block](https://developers.vtex.com/docs/apps/vtex.store-components/searchbar), exported by the Store Components app. Example:
```

| `area`                | `string` | Area where the banner will be displayed in the store. This needs to match the predefined area value in the banner configuration.                             | `undefined`   |
| `blockClass`          | `string` | Unique block ID to be used in [CSS customizations](https://developers.vtex.com/docs/guides/vtex-io-documentation-using-css-handles-for-store-customization). | `undefined`   |
| `horizontalAlignment` | `string` | Defines the horizontal alignment of the banner. Possible values are `left`, `center`, or `right`.                                                            | `center`      |

## App behavior

The Search app provides blocks that, when rendered as components, will improve the user's search experience in stores where the VTEX Intelligent Search engine is supported.

These components use `_q` as the query-string for the search term, meaning that if you want to track user searches in these components, you will need to add the `_q` query-string to the store's Google Analytics.

> Learn how to do this by reading our [Google Analytics search tracking](https://developers.vtex.com/docs/guides/vtex-io-documentation-setting-up-google-analytics-search-tracking) documentation.

Reviewer: @karenkrieger @Mariana-Caetano Vocês têm mais contexto sobre essa doc? Ela ainda faz sentido? Ela não está listada no nosso portal

Reviewer 2: Carol, essa doc do Google Analytics, não faz mais sentido. Ela é [relacionada ao Universal Analytics](https://support.google.com/analytics/answer/2649553?hl=en&ref_topic=6014102&sjid=4936669297032075139-SA), mas não tenho certeza qual seria a solução para esse app. @filipewl consegue nos ajudar aqui?
